migrations/sql/V2019.02.07.02.27__alter_expected_document_status.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE mine_expected_document DROP CONSTRAINT mine_expected_document_exp_document_status_guid_fkey;
```

Comment:
Will this leave orphaned `mine_expected_document` records?

Reply:
This is removing the foreign key relationship to the old mine document status table. I'm not sure how this would leave orphaned mine_expected_document records.

Reply:
I meant the file as a whole. I wasn't sure if removing the constraint without cleaning up the related records would lead to any issues.

Reply:
I don't see how it would orphan anything. If there are mine_expected_document records, they may lose any status details they had. the status table gets dropped completely, so nothing orphaned in the foreign table. I think we're good with this, as we should have no expected documents in production right now.